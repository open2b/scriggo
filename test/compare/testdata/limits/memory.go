// skip : enable before merging with 'master'.

// paniccheck -mem=10B

package main

func main() {
	s := [10]string{}
	for i := 0; i < 10; i++ {
		s[i] = "stringggg!!!"
	}
}
