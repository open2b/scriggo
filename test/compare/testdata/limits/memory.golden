panic: runtime error: out of memory: cannot allocate 158 bytes